proto/sentry_protos/snuba/v1/request_common.proto

```diff
@@ -16,6 +16,7 @@ message RequestMeta {
   TraceItemName trace_item_name = 7;
   bool debug = 10;
   string request_id = 11;
+  bool enable_trace_logs = 12;
```

Comment:
why is this not just the `debug` flag?

Reply:
Initially, I considered using enable_trace_logs to control both trace output capture and profile event logging. I have updated to use a single flag.